src/systems/ec/implementations/react/packages/media-container/MediaContainer.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const MediaContainer = ({
+  image,
+  alt,
+  sources,
+  tracks,
+  description,
+  className,
+  ...props
+}) => {
+  const classNames = classnames(className, 'ecl-media-container', {});
+
+  return (
+    <figure {...props} className={classNames}>
+      {Array.isArray(sources) && sources.length !== 0 ? (
+        /* eslint-disable-next-line jsx-a11y/media-has-caption */
+        <video
+          className="ecl-media-container__media"
+          controls="controls"
+          poster={image}
+          alt={alt}
+        >
+          {sources.map(source => (
+            <source key={source.src} src={source.src} type={source.type} />
+          ))}
+
+          {tracks.map(track => (
+            <track
```

Comment:
actually, React needs `srcLang` while the correct HTML attribute is `srclang`.

now, depending on your data, is you call your property `srcLang` and you can use `{...track}`, or you manually map `srcLang={track.srclang}`

either way, something needs to be done because right now, the property (in the data file) is called `srclang`, not `srcLang`